logging_manager/src/main.py

```diff
@@ -0,0 +1,45 @@
+import requests
+import time
+import glob
+import os
+
+# Call StorageManager
+exec(open("StorageManager.py").read())
+
+# Check if acknowledgment is received from StorageManager
+if "done" in globals():
+    storage_done = True
+
+# Check for Wi-Fi connection
+while True:
+    # Connected:
+    try:
+        response = requests.get("http://www.google.com", timeout=5)
+        response.raise_for_status()
+
+        # Call WiFiUploader
+        exec(open("WiFiUploader.py").read())
```

Comment:
Same here as w/ storage manager, import these python files, no need for shells here